cmd/config/validate.go

```diff
@@ -31,12 +37,30 @@ func NewValidateCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			c := config.GetCmdOpts()
 
-			loadingRules := config.ClientConfigLoadingRules{K0sVars: c.K0sVars}
```

Comment:
An (unfathomable hacky :see_no_evil:) alternative would be to just specify sth. like this:

```go
			loadingRules := config.ClientConfigLoadingRules{
				K0sVars: c.K0sVars,
				// HACK: the forward slash will never point to a valid file,
				// hence this effectively bypasses runtime config loading.
				RuntimeConfigPath: "/",
			}
```

That way the config loading code wouldn't be duplicated and the runtime config stuff would be disabled for this subcommand.

A slighty cleaner way would be to introduce another boolean field, say, `ignoreRuntimeConfig` to the `ClientConfigLoadingRules` struct to explicitly bypass the runtime config parts.